src/payer/routes.js

```diff
@@ -86,5 +86,23 @@ module.exports = [
       tags: tags,
       description: 'Get details based on correlationid'
     }
+  },
+  {
+      method: 'GET',
+      path: '/payerfsp/requests/{id}',
```

Comment:
Whats the difference between GET `/{fsp}/correlationid{id}` vs `/{fsp}/requests{id}` vs `/{fsp}/callbacks{id}` ?